app/assets/stylesheets/workspace.scss

```diff
@@ -1,6 +1,14 @@
+.workspaces-show {
+  .container-fluid {
+    margin: 0;
+    padding: 0;
+  }
+}
+
 #viewer {
+  clear: both;
   display: block;
-  height: 600px;
-  position: relative;
+  height: calc(100% - 55px); // should be $navbar-height, but variable is not available
```

Comment:
👍 I'll give that try.

I thought I was looking at the v4 variables file and saw $brandbar-height, but I was clearly mistaken.